core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -512,7 +512,7 @@ func (c *coordinator) filterUnfulfilledCallbacks(
 		aHeight := callbacksRequested[a].NextBeaconOutputHeight + callbacksRequested[a].ConfDelay.Uint64()
 		bHeight := callbacksRequested[b].NextBeaconOutputHeight + callbacksRequested[b].ConfDelay.Uint64()
 		if aHeight == bHeight {
-			return callbacksRequested[a].Callback.GasAllowance.Int64() < callbacksRequested[b].Callback.GasAllowance.Int64()
+			return callbacksRequested[a].GasAllowance < callbacksRequested[b].GasAllowance
```

Comment:
Was the type a big int before?

Reply:
Seems like it: https://github.com/smartcontractkit/chainlink/blob/d03d9772c62e648f54581f794837d13b15c27a74/core/gethwrappers/ocr2vrf/generated/vrf_coordinator/vrf_coordinator.go#L45-L54